Jenkinsfile.tab

```diff
@@ -0,0 +1,72 @@
+pipeline {
+  agent{
+    label 'BASIC_SLAVE&&!beefy_BASIC_SLAVE'
+  }
+
+  options {
+    timeout(time: 2, unit: 'HOURS')
+    timestamps()
+    ansiColor('xterm')
+  }
+
+  stages {
+    stage('Checkout') {
+        steps{
+            checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'a8c27fc2-52cd-499c-b648-0355ed5ca72a', url: 'https://github.com/webex/react-ciscospark']]]
+        }
+    }
+
+    stage('Install') {
+      steps {
+        withCredentials([
+          string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
+        ]) {
+          sh '''#!/bin/bash -ex
+            set +x
+            rm .nvmrc
+            curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
+            ~/.nvm/nvm.sh
+            echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' >> .npmrc
+            source ~/.bashrc || true
+            source ~/.profile || true'''
+          sh '''#!/bin/bash -ex
```

Comment:
Should this and the previous statement be combined into a single `sh` block?

Reply:
I can't remember why I had them separate for the portals jobs, i can try it.